uktrade/data-hub-frontend
# Migrate project edit history page to use new layout

## Description of change

The page for editing the team members for an investment project has been updated to use the new project layout.

The data route has been left in Express for now as changing this will require numerous changes to the `EditHistory` component itself (which is out of scope of this ticket).

## Test instructions

Go to the edit history page for an investment project. Everything should work as normal.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
